docs/pinterest/README.md

```diff
@@ -0,0 +1,53 @@
+<!-- markdownlint-disable -->
+
+# API Overview
+
+## Modules
```

Comment:
We do not need a lot of these modules. Can we create a delete list of sorts? For example, config md files, most utils, version md file are not needed

Reply:
I know some file looks useless but this supposed to work like dictionary, and it should have everything documented. But if we really don't want them to show up, we definitely can update the script